lib/bap_disasm/disasm.cpp

```diff
@@ -124,7 +124,7 @@ class disassembler {
         };
     }
 
-    void set_memory(int64_t addr, const char *data, int offset, int length) {
+    void set_memory(int64_t addr, const uint8_t *data, int offset, int length) {
```

Comment:
Understood. The reason I say this is because of the `ArrayRef<uint8_t>` parameter in `getInstruction`. I will correct this though.